models/gc_nnunet_pancreas/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM mhubai/base:latest
+
+# Specify/override authors label
+LABEL authors="sil.vandeleemput@radboudumc.nl"
+
+# Install PyTorch 2.0.1 (CUDA enabled)
+RUN pip3 install --no-cache-dir torch==2.0.1+cu118 -f https://download.pytorch.org/whl/torch_stable.html
+
+# Install git-lfs (required for downloading the model weights)
+RUN apt update && \
+    apt install -y --no-install-recommends git-lfs && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install the model weights and the algorithm files
+#  * Pull algorithm from repo into /opt/algorithm (main branch, commit e4f4008c6e18e60a79f693448562a340a9252aa8)
+#  * Remove .git folder to keep docker layer small
+#  * Replace input images path in process.py with an existing folder to avoid errors
+#  * Add specific data types and compression options to output data structures in process.py to reduce generated output footprint
+RUN git clone https://github.com/DIAGNijmegen/CE-CT_PDAC_AutomaticDetection_nnUnet.git /opt/algorithm && \
+    cd /opt/algorithm && \
+    git reset --hard e4f4008c6e18e60a79f693448562a340a9252aa8 && \
+    rm -rf /opt/algorithm/.git && \
+    sed -i 's/Path("\/input\/images\/")/Path("\/app")/g' /opt/algorithm/process.py && \
```

Comment:
Manipulating code inside the Dockerfile goes against our transparency efforts.

As we already use a cli (`utils/cli.py`) to wrap the calls to the python class, can we not just overwrite the class variables from there?

After importing `PDACDetectionContainer`,  from my understanding you, for example, could just set `PDACDetectionContainer.ct_ip_dir = Path("\/app")`.

Reply:
I agree, we discussed this with Natália. We will create a PR with the code changes to the original repository. If that is accepted we will remove the manual code patches.

Reply:
Created a PR for this: https://github.com/DIAGNijmegen/CE-CT_PDAC_AutomaticDetection_nnUnet/pull/5

Reply:
Updated to the latest algorithm/model commit with the merged PR